### ISSUE TITLE
bug report

### ISSUE THREAD

**Issue by shawn-yee** [johnsoncodehk/volar#48]
I often get this error, even without writing any code.
It has occasionally appeared in previous and the latest version.

Volar-API
Request textDocument/codeLens failed.
  Message: Request textDocument/codeLens failed with message: Invalid count value
  Code: -32603

**Comment by johnsoncodehk** [comment]
hi @shawn-yee, can you post your tsconfig?

**Comment by shawn-yee** [comment]
My code is automatically generated via vue-cli's typescript template.
It would be nice if you could provide a example template in  the repo.

```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "esnext",
    "strict": true,
    "jsx": "preserve",
    "importHelpers": true,
    "moduleResolution": "node",
    "skipLibCheck": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "sourceMap": true,
    "baseUrl": ".",
    "types": [
      "webpack-env"
    ],
    "paths": {
      "@/*": [
        "src/*"
      ]
    },
    "lib": [
      "esnext",
      "dom",
      "dom.iterable",
      "scripthost"
    ]
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.tsx",
    "src/**/*.vue",
    "tests/**/*.ts",
    "tests/**/*.tsx"
  ],
  "exclude": [
    "node_modules"
  ]
}

**Comment by johnsoncodehk** [comment]
I try this tsconfig and vue-cli typescript template but no luck. I need a repro case. (I know this is bother, but thanks if you can provide :P)

**Comment by johnsoncodehk** [comment]
close because no repro case, if you can provide repro case I will check it again.

**Comment by notiv-nt** [comment]
@johnsoncodehk 

https://user-images.githubusercontent.com/10743009/109144010-b795f500-7771-11eb-9f34-64090938cd4d.mp4

A new window, empty document, vue syntax

**Comment by johnsoncodehk** [comment]
@notiv-nt I think this is not a same problem, but I will fix it, thanks for report!

**Comment by johnsoncodehk** [comment]
> @johnsoncodehk
> 
>  Screen.Recording.2021-02-25.at.1.56.49.PM.mp4 
> A new window, empty document, vue syntax

fixed in 0.21.17.
